tests/ffi/agent-flood/test.sh

```diff
@@ -64,15 +64,43 @@ run_test_containers(){
     done
 }
 
+# Extract the Network mode from qm quadlet file
+get_qm_network_mode(){
+    qm_config_file="/usr/share/containers/systemd/qm.container"
+
+    # Check if the configuration file exists
+    if [ ! -f "$qm_config_file" ]; then
+        echo "Configuration file not found: $qm_config_file"
+        exit 1
+    fi
+
+    # Extract the Network using awk
+    qm_config_file="/usr/share/containers/systemd/qm.container"
+    qm_network_mode=$(awk -F'=' '/Network=private/ { print $2 }' "$qm_config_file")
+    echo "${qm_network_mode}"
+}
+
 disk_cleanup
 prepare_test
 reload_config
 prepare_images
 
+# Assign value to ${controller_host_ip} according to qm network mode
+controller_host_ip="localhost"
+if [ "$(get_qm_network_mode)" == "private" ]; then
```

Comment:
so what are the option here? 
private| host | .. please use `man podman run`

What happens in the else case? 
Does test exit with failure?

Reply:
Hi @Yarboa , thanks for the review.
Valid values for the container's network mode are:

- bridge, 
-  network  name  or ID>
- none
- container:id
- host
- ns
- private
- slirp4netns
- pasta

May I ask what is the bluechi controller host ip corresponding to these modes?

Reply:
Only option as per RA is network private for qm.
So the others should fail the test with error message,

Reply:
done